right/src/config/parse_keymap.c

```diff
@@ -4,6 +4,16 @@
 
 #define longCompactLengthPrefix 0xff
 
+enum {
```

Comment:
Please typedef this enum as `parser_error_t`, move it to `parse_keymap.h`, and use `parser_error_t` instead of `uint8_t` wherever it makes sense.